src/templates/ResourceAccessRequests.ts

```diff
@@ -12,34 +13,34 @@ export const ACCESS_LEVELS = {
 class ResourceAccessRequests extends BaseService {
   protected ACCESS_LEVELS: typeof ACCESS_LEVELS;
 
-  constructor(resourceType, baseParams) {
+  constructor(resourceType: string, baseParams: BaseModelContructorOptions) {
     super(baseParams);
 
     this.url = URLJoin(this.url, resourceType);
     this.ACCESS_LEVELS = ACCESS_LEVELS;
   }
 
-  all(resourceId) {
+  all(resourceId: string) {
     const rId = encodeURIComponent(resourceId);
 
     return RequestHelper.get(this, `${rId}/access_requests`);
   }
 
-  request(resourceId) {
+  request(resourceId: ResourceId) {
     const rId = encodeURIComponent(resourceId);
 
     return RequestHelper.post(this, `${rId}/access_requests`);
   }
 
-  approve(resourceId, userId, { accessLevel = 30 }) {
+  approve(resourceId: ResourceId, userId: UserId, { accessLevel = 30 }) {
     const [rId, uId] = [resourceId, userId].map(encodeURIComponent);
 
     return RequestHelper.post(this, `${rId}/access_requests/${uId}/approve`, {
-      accessLevel,
+      access_level: accessLevel,
```

Comment:
Seems like a mistake when we have https://github.com/jdalrymple/node-gitlab/blob/f946569882bdba81bda1e8e34c1c7c5d968fbd70/src/infrastructure/RequestHelper.ts#L46-L55